app/players/elves/druid.py

```diff
@@ -0,0 +1,18 @@
+from .elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 favourite_spell: str,
+                 ) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability

Reply:
The comment is not implemented